CONTRIBUTING.md

```diff
@@ -87,6 +87,18 @@ This repository follows the [PSR-2 standard](http://www.php-fig.org/psr/psr-2/)
 you must follow these rules.
 
 
+## Coding style
+
+Apart from the standards above, we have a few more rules which we think makes programming more logical:
+
+1. Don't align key value pairs: it messes up diff (this applies to code, config, even docblocks)
+2. Use trailing commas after last array elements
+
+See these articles for reference:
```

Comment:
May be worth to have the deciding worklfow for new rules in the documentation WDYT ?